crates/hotshot/src/traits/networking/libp2p_network.rs

```diff
@@ -268,53 +267,19 @@ impl<TYPES: NodeType> TestableNetworkingImplementation<TYPES>
                 )
                 .expect("Failed to sign DHT lookup record");
 
-                // we want the majority of peers to have this lying around.
+                // We want 2/3 of the nodes to have any given record in the DHT
                 let replication_factor = NonZeroUsize::new(2 * expected_node_count / 3).unwrap();
```

Comment:
Should this be `div_ceil` maybe? To never fall on the side of < 2/3

Reply:
Sure, good idea